test/sequential/test-watch-mode.mjs

```diff
@@ -104,7 +104,7 @@ async function runWriteSucceed({
   if (watchFlag !== null) args.unshift(watchFlag);
   const child = spawn(execPath, args, { encoding: 'utf8', stdio: 'pipe', ...options });
   let completes = 0;
-  let cancelRestarts = () => {};
+  let cancelRestarts = () => { };
```

Comment:
These seem to have changed, however when running `make-lint-js` or `make-lint-js-fix`, I dont see any errors?